CONTRIBUTING.md

```diff
@@ -1,11 +1,91 @@
-Contributing to RDMO
-====================
+# Contributing to RDMO
```

Comment:
I added a few sections to your contributor guide, based on the hypermodern python cookiecutter template. What do you think?

Reply:
Done.